tests/overhead/docker-compose.yml

```diff
@@ -55,7 +54,7 @@ services:
       - DD_TRACE_NO_AUTOLOADER=true
     volumes:
       - ./callgrind-files:/var/www/callgrind-files
-      # - ../../:/dd-trace-php
+      - ../../:/dd-trace-php
```

Comment:
Can we avoid mounting the extension dir by default? Then we can mount while working on something specific before the final measure. wdyt?